server/src/main/java/com/broadcom/lsp/cobol/core/semantics/SemanticContext.java

```diff
@@ -29,15 +30,18 @@
  * variables, paragraphs, copybooks etc., with their definitions and usages.
  */
 @Value
+@Builder
 @AllArgsConstructor
```

Comment:
@AllArgsConstructor can be removed, added as a part of @Value

Reply:
The builder overrides constructor to package-private `@AllArgsConstructor`, so this constructor can be used only inside a package :-(